modules/misunderstood/src/hooks/after_incoming_middleware/00_misunderstood_flag.js

```diff
@@ -1,27 +1,43 @@
 const axios = require('axios')
 const _ = require('lodash')
 
-const flag = async () => {
-  const matchesQuickReply = async () => {
-    const { currentFlow } = event.state.context
-    if (currentFlow && currentFlow.startsWith('skills/choice')) {
-      const flow = await bp.ghost.forBot(event.botId).readFileAsObject('flows', currentFlow)
-      const preview = event.preview.toLowerCase()
-      const kw = _.chain(flow.skillData.keywords)
-        .values()
-        .flatten()
-        .find(kw => kw.toLowerCase() === preview.toLowerCase())
-        .value()
-
-      if (kw) {
-        return true
-      }
+const { currentFlow, currentNode } = event.state.context
+
+const matchesQuickReply = async flow => {
+  if (currentFlow.startsWith('skills/choice')) {
```

Comment:
Should be flow instead of currentflow... but should be good since the check is done earlier